src/mbgl/renderer/sources/render_geojson_source.cpp

```diff
@@ -166,7 +166,7 @@ RenderGeoJSONSource::queryFeatureExtensions(const Feature& feature,
         return {};
     }
 
-    const auto clusterID = getProperty<uint64_t>(feature.properties, "cluster_id");
+    const auto clusterID = getProperty<int64_t>(feature.properties, "cluster_id");
```

Comment:
cluster_id is a uint64_t as created by https://github.com/mapbox/supercluster.hpp/blob/master/include/supercluster.hpp#L46 and is also referenced like this in our testing https://github.com/mapbox/mapbox-gl-native/blob/master/test/api/query.test.cpp#L196.